enterprise/cmd/frontend/internal/codeintel/init.go

```diff
@@ -65,5 +65,11 @@ func Init(ctx context.Context, enterpriseServices *enterprise.Services) error {
 		return codeintelhttpapi.NewUploadHandler(store, bundleManagerClient, internal)
 	}
 
+	newCodeIntelInternalProxyHandler, err := makeInternalProxyHandlerFactory()
```

Comment:
```suggestion
	h, err := newInternalProxyHandler()
```

1. `makeInternalProxyHandlerFactory` sounds like this function makes a factory which produces internal proxy handlers - but that is not the case.
2. `newCodeIntelInternalProxyHandler` does not need `codeintel` in it - you are in the codeintel package there is no need for such long var names I think.